src/components/FoodComponent/FoodComponent.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import { actionClear, actionFoodNutrients } from "../../actions"
+import { connect } from "react-redux";
+// import styles from "./Food.module.scss";
+// import { Link } from "react-router-dom";
+
+
+class FoodComponent extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {  };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick() {
+    this.props.dispatchClear();
+    this.props.dispatchFoodNutrients(this.props.fdcId);
```

Comment:
Would it be better to have the calories show (instead of fdcId) under the description?